x-pack/plugins/infra/common/http_api/inventory_views/v1/find_inventory_view.ts

```diff
@@ -5,28 +5,9 @@
  * 2.0.
  */
 
-import { nonEmptyStringRt } from '@kbn/io-ts-utils';
 import * as rt from 'io-ts';
-
-export const findInventoryViewAttributesResponseRT = rt.strict({
```

Comment:
**question**: I remembered this was a strict type to strip out the non-essential properties for this endpoint when the response is encoded in the server, I think doing so it will now leak all SO attributes on this response for every saved view, could you verify that please?

Reply:
I didn't see any weird attributes. I'll check it again.